src/main/java/org/uniovi/i3a/incimanager/web/WebController.java

```diff
@@ -1,92 +0,0 @@
-/*
- * This source file is part of the web-service open source project.
- *
- * Copyright (c) 2018 willy and the web-service project authors.
- * Licensed under GNU General Public License v3.0.
- *
- * See /LICENSE for license information.
- * 
- */
-package org.uniovi.i3a.incimanager.web;
-
-import java.util.HashMap;
-import java.util.LinkedList;
-import java.util.List;
-import java.util.Map;
-
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.boot.autoconfigure.domain.EntityScan;
-import org.springframework.http.HttpStatus;
-import org.springframework.stereotype.Controller;
-import org.springframework.validation.BindingResult;
-import org.springframework.web.bind.annotation.ModelAttribute;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RequestMethod;
-import org.uniovi.i3a.incimanager.rest.AgentsQueryFormatter;
-import org.uniovi.i3a.incimanager.rest.AgentsConnection;
-import com.mashape.unirest.http.HttpResponse;
-import com.mashape.unirest.http.JsonNode;
-
-/**
- * Instance of WebLoginController.java
- * 
- * @author
- * @version
- */
-@EntityScan
-@Controller
-public class WebController {
```

Comment:
I think web controller is also compulsory, please, before making a PR update your brach.